Default.sublime-keymap

```diff
@@ -308,6 +317,15 @@
         ]
     },
     {
+        "keys": ["k"],
+        "command": "gs_status_navigate_file",
+        "args": { "forward": false },
+        "context": [
+            { "key": "setting.git_savvy.status_view", "operator": "equal", "operand": true },
+            { "key": "setting.git_savvy.vintageous_friendly", "operator": "equal", "operand": true }
+        ]
+    },
+    {
```

Comment:
Does Vintagious use [command mode](http://docs.sublimetext.info/en/latest/reference/key_bindings.html#command-mode)?  That might be a more robust determinant for these key-binding contexts.